playground/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/redhat-et/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
+	wget -P models https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -o /dev/null
+	# curl -s -S -L -f https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -z $@ -o $@.tmp && mv -f $@.tmp $@ 2>/dev/null || rm -f $@.tmp $@
```

Comment:
I see this is implementing the "transactional write" pattern, does `wget` actually do that by default?  It doesn't seem so.  Curious why wget over curl.

Reply:
The curl command is preferred but it is broken on GHA. I'll try to fix it.